requirements.txt

```diff
@@ -1,6 +1,8 @@
 # Runtime requirements for the python 3 version of cirq.
 
 dataclasses; python_version < '3.7'
+enum34; python_version < "3.4"
+google-api-core[grpc] >= 1.14.0, < 2.0.0dev
```

Comment:
How much of an overhead is this?  I am (slightly) concerned about pulling in a bunch of google stuff for people who are using cirq for non-engine related tasks.

Reply:
Thanks, I forgot to actually remove the google-api-python-client dependency.

Fair. I'm not certain how to qualify the overhead. It definitely adds "unneeded" dependencies for people who are not using the engine. We talked about not requiring this on install and only complaining at runtime if a user attempted to create an engine or having the client as a new optional module but settled on keeping it in cirq for now.